samples/dnn/yolo_object_detection.cpp

```diff
@@ -1,3 +1,23 @@
+//! Brief Sample of using OpenCV dnn module in real time with device capture, video and image.
+//! Questions and suggestions email to: Alessandro de Oliveira Faria cabelo@opensuse.org or blog http://assuntonerd.com.br
+/*
+ * VIDEO DEMO:
+ * https://www.youtube.com/watch?v=NHtRlndE2cg
```

Comment:
It is preferred to put video and usage hints into corresponding tutorials (see here example for [GoogLeNet](https://docs.opencv.org/master/d5/de7/tutorial_dnn_googlenet.html)). We prefer to avoid large comments headers in examples.
See source code of this tutorial here: https://github.com/opencv/opencv/tree/3.3.1/doc/tutorials/dnn/dnn_googlenet

BTW, Don't forget to add created tutorial into TOC (entry includes name of tutorial's author): https://github.com/opencv/opencv/blob/3.3.1/doc/tutorials/dnn/table_of_content_dnn.markdown

Documentation snapshots are available for PRs too (see "preview" artifacts of "Docs" builder) - for this PR link is http://pullrequest.opencv.org/buildbot/export/pr/10207/docs/

Reply:
ok, thanks

Reply:
ok now? 
https://github.com/cabelo/opencv/blob/cabelo-opencv/doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown